applications/asset_tracker_v2/src/ext_sensors/Kconfig

```diff
@@ -16,6 +16,59 @@ config EXTERNAL_SENSORS_ACTIVITY_DETECTION_AUTO
 	  If the option is disabled, the accelerometer trigger handler must manually be enabled to
 	  detect activity.
 
+config EXTERNAL_SENSORS_BME680_BSEC
+	bool "Use Bosch BME680 library"
+	depends on !BME680
+	help
+	  Enable the use of Bosch BSEC library. The library itself is not a part of
+	  NCS and must be downloaded from Bosch Sensortec and placed in the path referenced to by
+	  the EXTERNAL_SENSORS_BME680_BSEC_PATH. This configuration depends on the BME680 zephyr
```

Comment:
Nit: Not quite sure what's the convention, but this could be prefixed with `CONFIG_` to avoid any confusion when users try to add it to their `prj.conf`